charts/hivemq-platform-operator/templates/deployment.yml

```diff
@@ -157,11 +157,19 @@ spec:
             timeoutSeconds: 10
           resources:
             limits:
-              cpu: {{.Values.resources.cpu | quote}}
-              memory: {{.Values.resources.memory | quote}}
+              cpu: {{ include "hivemq-platform-operator.resources-limits-cpu" . }}
+              memory: {{ include "hivemq-platform-operator.resources-limits-memory" . }}
+              {{- if and .Values.resources.overrideLimits .Values.resources.overrideLimits.ephemeralStorage }}
+              ephemeral-storage: {{ .Values.resources.overrideLimits.ephemeralStorage | quote }}
```

Comment:
Does it make sense to add a helper function for ephemeral-storage here as well?

Reply:
I initially added a helper function as well, but the problem here is with the templating language itself... cause in this particular case since this field is optional, we need to wrap the whole `ephemeral-storage` inside the helper function but this creates a lot unnecessary whitespaces and newlines, making the section quite difficult and cumbersome to handle... so I decided to leave this one without any helper function.
I will try to give it another look to see if I can workaround this somehow.

Reply:
Managed to do it in a helper function finally 😀